README.md

```diff
@@ -1,95 +1,80 @@
-# demo-package
+# F2F CRI 
 
-This project contains source code and supporting files for a serverless application that you can deploy with the SAM CLI. It includes the following files and folders. Test
+## Installation
 
-- HelloWorldFunction/src/main - Code for the application's Lambda function.
-- events - Invocation events that you can use to invoke the function.
-- HelloWorldFunction/src/test - Unit tests for the application code.
-- template.yaml - A template that defines the application's AWS resources.
+Recommended global installed packages: 
 
-The application uses several AWS resources, including Lambda functions and an API Gateway API. These resources are defined in the `template.yaml` file in this project. You can update the template to add AWS resources through the same deployment process that updates your application code.
+- nodejs
+- nvm
+- python3
+- pre-commit (installed via pip3)
```

Comment:
`sam` is not listed as a dependency here, but we're using it below to locally invoke lambdas. Does it need adding in?